install.sh

```diff
@@ -51,6 +50,37 @@ echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output
 # Skip all checks if running on a docker container.
 [[ -f "/.dockerenv" ]] && CREW_FORCE_INSTALL=1
 
+if [[ "$CREW_FORCE_INSTALL" ]]; then
+  echo_info "In container, so skipping hardware checks"
+elif tty | grep -q /dev/pts1; then
+  echo_success "You are in VT-2."
+  if [ "${EUID}" == "0" ]; then
```

Comment:
How about if `${USER} != 'chronos'`?

Reply:
We could do that, but then I think there is another PR playing around with using this NOT on chromeos... and the existing code is simpler...

Reply:
To be fair, my PR isin't dealing with actually installing crew-- just being able to run some things in `tools/`.

But, I might look into that in future, and it makes sense not to throw up additional roadblocks.